nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/StandardTlsConfiguration.java

```diff
@@ -203,6 +205,38 @@ public static StandardTlsConfiguration fromNiFiProperties(NiFiProperties niFiPro
         return tlsConfiguration;
     }
 
+    /**
+     * Returns a {@link org.apache.nifi.security.util.TlsConfiguration} instantiated from the relevant {@link NiFiProperties} properties.
+     *
+     * @param niFiProperties the NiFi properties, as a simple java.util.Properties object
+     * @return a populated TlsConfiguration container object
+     */
+    public static StandardTlsConfiguration fromNiFiProperties(Properties niFiProperties) {
```

Comment:
Recommend changing the return type to `TlsConfiguration` to promote use the interface instead of the implementation class.
```suggestion
    public static TlsConfiguration fromNiFiProperties(final Properties niFiProperties) {
```